website/static/js/profile.js

```diff
@@ -468,19 +480,35 @@ var SocialViewModel = function(urls, modes) {
     var self = this;
     BaseViewModel.call(self, urls, modes);
     TrackedMixin.call(self);
+    var i;
 
     self.addons = ko.observableArray();
-
-    self.personal = extendLink(
-        // Note: Apply extenders in reverse order so that `ensureHttp` is
-        // applied before `url`.
-        ko.observable().extend({
-            trimmed: true,
-            url: true,
-            ensureHttp: true
-        }),
-        self, 'personal'
-    );
+  
+    self.profileWebsites = ko.observableArray();
+                
+    self.hasProfileWebsites = ko.computed(function() {
```

Comment:
I'm not sure that this is necessary. Only non-empty, valid values should ever be allowed the `profileWebsites()` array. If that's the case, `hasProfileWebsites` is unnecessary because you could just check `if (self.profileWebsites())`.